runtime-modules/content-working-group/src/lib.rs

```diff
@@ -2033,6 +2041,41 @@ decl_module! {
             let mint = <minting::Module<T>>::mints(mint_id); // must exist
             let new_capacity = mint.capacity() + additional_capacity;
             let _ = <minting::Module<T>>::set_mint_capacity(mint_id, new_capacity);
+
+            Self::deposit_event(RawEvent::MintCapacityIncreased(
+                mint_id, additional_capacity, new_capacity
+            ));
+        }
+
+        /// Sets the capacity of the current active mint
+        pub fn set_mint_capacity(
+            origin,
+            new_capacity: minting::BalanceOf<T>
+        ) {
+            ensure_root(origin)?;
+
+            let mint_id = Self::mint();
+
+            // Mint must exist - it is set at genesis
+            let mint = <minting::Module<T>>::mints(mint_id);
+
+            let current_capacity = mint.capacity();
+
+            if new_capacity != current_capacity {
+                // Cannot fail if mint exists
+                let _ = <minting::Module<T>>::set_mint_capacity(mint_id, new_capacity);
```

Comment:
Same as above

Reply:
done in 1d115a84a74548faa113f678570de987e90417b4